src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -1667,6 +1693,49 @@ protected override void HandleModulesRequestAsync(IRequestResponder<ModulesArgum
             responder.SetResponse(response);
         }
 
+        // test -- need to delete
+        protected override void HandleDisassembleRequestAsync(IRequestResponder<DisassembleArguments, DisassembleResponse> responder)
+        {
+            DisassembleResponse response = new DisassembleResponse();
+
+            int hr;
+            DisassembleArguments disassembleArguments = responder.Arguments;
+            Debug.Assert(!string.IsNullOrEmpty(disassembleArguments.MemoryReference));
+            try
+            {
+                IDebugMemoryContext2 memoryContext = GetMemoryContext(disassembleArguments.MemoryReference, disassembleArguments.Offset, out ulong address, out hr);
```

Comment:
>hr [](start = 160, length = 2)

Throw if hr is failed?

Reply:
I assume I will need to create a new error string in AD7Resources. Is this correct?

Reply:
Yes